daal4py/sklearn/_utils.py

```diff
@@ -17,26 +17,20 @@
 
 import numpy as np
 
-from daal4py import __daal_link_version__, __daal_run_version__
+from daal4py import __daal_link_version__ as dv, __has_dist__
 
-def daal_check_version(trad_target, dpcpp_target = None):
-    daal_run_version = tuple(map(int, (__daal_run_version__[0:4], __daal_run_version__[4:8])))
-    daal_link_version = tuple(map(int, (__daal_link_version__[0:4], __daal_link_version__[4:8])))
-    result = True
-    if daal_run_version[1] > 100:
-        # daal dpcpp
-        result = result and daal_run_version >= dpcpp_target if dpcpp_target is not None else False
+def daal_check_version(rule):
+    target = tuple(map(int, (dv[0:4], dv[4:8], dv[8:9])))
```

Comment:
dv[0:4] = major version
dv[4:8] = minor version
dv[8:9] = update + status ?

I would write here as variables for the reader to understand. And how do we ultimately compare status?

Reply:
The initial format is 20210110_20200928 and i've change it to 20210110**B**_20200928

I would personally left this as is for now - because it's current version. Will add comments for now.